extensions/liquibase/deployment/src/main/java/io/quarkus/liquibase/LiquibaseProcessor.java

```diff
@@ -54,26 +55,21 @@
 
     private static final Logger LOGGER = Logger.getLogger(LiquibaseServiceLoader.class);
 
-    /**
-     * The change log parser factory singleton used for the liquibase service classpath scanner
-     */
-    private static final ChangeLogParserFactory CHANGE_LOG_PARSER_FACTORY = ChangeLogParserFactory.getInstance();
```

Comment:
This is where all the time is passed: it has a very flawed and slow scanning process.

We now only create that instance when building a native image.

Reply:
Hi - I am the Liquibase Community Manager. Wanted to let y'all know that this is something that should be improved in our next major release, 4.0.0. Keep an eye on the blog for more info. 4.0 Beta 1 should be available in the next month.

Reply:
@SteveDonie ah cool! Feel free to ping us if you want to discuss things.